wordpress/wp-content/plugins/memberful-wp/src/acl/term_options.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+function memberful_wp_get_term_available_to_any_registered_users( $term_id ) {
+  return get_term_meta( $term_id, 'memberful_available_to_any_registered_user', TRUE ) === "1";
+}
```

Comment:
I think the function's name is a bit misleading as it doesn't get a term available to any user but instead checks if the given term is available to any user. Not sure if there's a convention for this in PHP land, but maybe `memberful_wp_is_term_available_to_any_registered_users`?

Now that I thought about it some more, I think I understand the intention: to get the term's column `available_to_any_registered_user`. But I think it's a bit ambiguous, imo.